src/transformers/integrations.py

```diff
@@ -667,6 +667,27 @@ def __init__(self):
         else:
             self._log_model = os.getenv("WANDB_LOG_MODEL", "false").lower()
 
+    def get_env_vars(self):
```

Comment:
@nbroad1881: Many of these are WANDB [ENVIRONMENT_VARIABLES](https://docs.wandb.ai/guides/track/advanced/environment-variables) and are auto detected by wandb upon `wandb.init`. We should be able to keep the `WandbCallBack` environment variables different from the wandb environment variables which work even when not passed explicitly to `wandb.init`

Reply:
Which ones are not auto detected?